tests/conftest.py

```diff
@@ -7521,3 +7521,59 @@ def factory(bucketclass_dict=None):
         return mockup_logger, source_bucket, target_bucket
 
     return factory
+
+
+@pytest.fixture(scope="session")
+def enable_rbd_metrics(request):
+    ct_pod = pod.get_ceph_tools_pod()
```

Comment:
shouldn't we add a condition for external mode only?

Reply:
good question. Even though the fixture returns back the values of exclude_perf_counters and rbd_stats_pools it may cover regression bug, when these values are not configured by default. 
I will add skip for this fixture for non-external mode clusters.